CocoaLumberjack.podspec

```diff
@@ -2,7 +2,7 @@
 Pod::Spec.new do |s|
 
   s.name     = 'CocoaLumberjack'
-  s.version  = '2.3.0'
+  s.version  = '2.4.0'
```

Comment:
Sure. I just saw that adding support for Swift 2.2 also lead to a minor version increase. I'll change this to 3.0.0 (kinda matches the Swift version 😄)